### ISSUE TITLE
Using relative paths for imported types

### ISSUE THREAD

**Issue by DEVTomatoCake** [futurGH/ts-to-jsdoc#43]
Nice project!

Is there any way of making it use relative paths instead of absolute ones for imported types when using the project option? Absolute paths makes a project mostly unable to collaborate to together with others with different setups.

There doesn't seem to be any disadvantage in using e.g.
```ts
@returns {import("./file").File}
```
over
```ts
@returns {import("C:/path/to/file").File}
```

**Comment by futurGH** [comment]
Hey! Can you share a reproduction where this happens?

**Comment by DEVTomatoCake** [comment]
```ts
static contextmenu=new Contextmenu("guild menu");
```
gets transformed to
```js
/**
 * @static
 * @default import("C:/path/to/folder/contextmenu").Contextmenu
 */
static contextmenu = new Contextmenu("guild menu");
```

I'm trying this on [MathMan05/JankClient](https://github.com/MathMan05/JankClient), the above is an example from `guild.js` after running `npx ts-to-jsdoc -p ./tsconfig.json`. I'd guess it happens to other projects too though.

**Comment by futurGH** [comment]
Thanks, fixed in 2.1.2!
